modules/ROOT/pages/provisioning-aws.adoc

```diff
@@ -102,7 +102,36 @@ ignition:
       source: s3://instance1-infra/bootstrap.ign
 ----
 
-Then you can launch the instance using the same command as xref:#_customized_example[], passing the minimal configuration you just created.
+Then you can launch the instance using the following command, passing the minimal configuration you just created.
+
+Note: You need to create an IAM profile with `s3:GetObject` permission, and attach the role to the instance.
```

Comment:
Let's add here the command to do that as well

Reply:
Add an URL here about how to create an IAM profile, WDYT?